ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -23,6 +23,7 @@ set TUNE_ERR ML<<CP4CP4CP4CP4CP4
 #
 # Try to mount the microSD card.
 #
+# REBOOTWORK this needs to start after the flight control loop
```

Comment:
I plan to speed up in-air reboot, and these comments mark locations to be reviewed.